### DISCUSSION TITLE
Can't login | Unraid Docker

### DISCUSSION THREAD

**Discussion by supawiz6991** [alexjustesen/speedtest-tracker#980] [Q&A] [answered]
Hi All, 
I installed the docker in Unraid along side MariaDB. I can get the container to load and get to the login page but when I click submit it sends me back to the getting started page and I can't seem to do anything else from there (aside from following the docs and other links).
Any ideas?

**Comment by alexjustesen** [comment, marked as answer]
Fixed in `v0.14.0-beta6`, you can also just go right to `/admin` to get to the panel.

**Comment by supawiz6991** [comment]
Thanks! /admin worked so I'll use that for now.
